unlock-app/src/_server.js

```diff
@@ -27,6 +27,13 @@ function _server(port, dev) {
         } else if (path === 'demo') {
           const params = route('/demo/:lockaddress')(pathname)
           app.render(req, res, '/demo', Object.assign(params, query))
+        } else if (path === 'blog') {
+          const params = route('/blog/:slug')(pathname)
+          if (params.slug) {
```

Comment:
same - you can just render the blog and let the component parse the slug out and route there